test/rekt/features/parallel/features.go

```diff
@@ -48,12 +49,8 @@ func ParallelWithTwoBranches() *feature.Feature {
 	eventBody := `{"msg":"test msg"}`
 	event.SetData(cloudevents.ApplicationJSON, []byte(eventBody))
 
-	channelTemplate := parallel.ChannelTemplate{
-		TypeMeta: v1.TypeMeta{
-			APIVersion: "messaging.knative.dev/v1",
-			Kind:       "InMemoryChannel",
-		},
-	}
+	channelTemplate := channel_template.ImmemoryChannelTemplate()
```

Comment:
Maybe on a different follow up PR would be great to pass this as parameter as well.